src/BBLibc/BBLibc.cpp

```diff
@@ -0,0 +1,88 @@
+#include <crtdbg.h>
+#include "direct.h"
+
+#include "MessageManager.h"
+#define BBLIBC_LIB_EXPORT

```

Comment:
If I put `#include "MessageManager.h"` below the `BBLIBC_LIB_EXPORT` it won't compile. Because `class B_MessageManager mout" (?mout@@3VB_MessageManager@@A)` already defined in the `"MessageManager.cpp"`

Reply:
And even `#include "direct.h"` should go below?

Reply:
I see.